docs/docs/self-host/self-host-lightdash-restack.mdx

```diff
@@ -0,0 +1,74 @@
+# Self-Host Lightdash with Restack
+
+Restack is a managed service for deploying and managing open-source software.
+
+## Deploy to AWS with Restack
+
+To deploy Lightdash to your own AWS with Restack:
+
+1. [Sign up for a Restack account](#sign-up-for-a-restack-account).
+2. [Add AWS credentials with AdministratorAccess](#add-aws-credentials-with-administratoraccess).
+3. [One-click cluster creation with Restack](#one-click-cluster-creation-with-restack).
+4. [Deploy Lightdash on Restack](#deploy-lightdash-on-restack).
+5. [Start using Lightdash](#start-using-lightdash).
+6. [Deploy multiple instances of Lightdash](#deploy-multiple-instances-of-lightdash).
+
+### Sign up for a Restack account
+
+To Sign up for a Restack account, visit [www.restack.io/signup](https://www.restack.io/signup). You can sign up with your corporate email address or your GitHub profile. You do not need a credit card to sign up.
+![restack-signup](../assets/restack-sign-up.png)
+If you already have an account, go ahead and login to Restack at [www.restack.io/login](https://www.restack.io/login).
+
+### Add AWS credentials with AdministratorAccess
+
+To deploy Lightdash in your own AWS infrastructure with Restack, you will need to add your credentials as the next step.
+
+Make sure that this account has *AdministratorAccess*. This is how Restack can ensure an end-to-end cluster creation and cluster management process.
+
+1. Navigate to *Clusters* in the left-hand navigation menu.
+2. Select the *Credentials* tab.
+3. Click *Add credential*.
+   ![add-credentials](../assets/restack-add-credentials.png)
+4. Give a suitable title to your credentials for managing them later.
+5. Enter your *AWS Access Key ID* and *AWS Secret Access key*.
+6. Click *Add credential*.
+   ![add-aws-credentials](../assets/restack-add-AWS-creds.png)
+
+>[How to get your AWS Access key ID and AWS Secret Access Key](https://docs.aws.amazon.com/accounts/latest/reference/root-user-access-key.html)
+
+### One-click cluster creation with Restack
+
+:::tip
+Why do I need a cluster?<br/>
```

Comment:
What's are all these <br/> ? :suspect: did chatgpt write this for you ?

Reply:
This is already in the docs! This was contributed by the team at Restack, I believe it's a template set of docs they use. I'm happy to keep it in